src/core/database/highlight-cypher.util.ts

```diff
@@ -9,7 +9,15 @@ export function highlight(query: string) {
     padding: 0,
     theme,
   });
-  pretty = pretty.split('\n').slice(0, -1).join('\n');
+  pretty = pretty
+    .split('\n')
+    .slice(0, -1)
+    // strip off trailing spaces meant for a "code block background."
+    // we don't set a background color, and this makes copy/pasting into
+    // neo4j editor painful.
+    // eslint-disable-next-line no-control-regex
+    .map((line) => line.replace(/( )+\u001B\[39m$/, ''))
```

Comment:
What is this pattern matching for here that a .trim() wouldn't work for?

Reply:
Trim doesn't work because the spaces are followed by that terminal escape sequence. I tried lol